docs_sphinx/introduction/brian1_to_2/preferences.rst

```diff
@@ -0,0 +1,51 @@
+Preferences (Brian 1 --> 2 conversion)
+========================================
+.. sidebar:: Brian 2 documentation
+
+    For the main documentation about preferences, see the document
+    :doc:`../../advanced/preferences`.
+
+In Brian 1, preferences were set either with the function ``set_global_preferences`` or by creating a module
+somewhere on the Python path called ``brian_global_config.py``.
+
+
+Setting preferences
+-------------------
+
+There is now no function ``set_global_preferences``. Instead, importing from ``brian2`` gives you a variable
```

Comment:
"There is now no function" sounds odd to my ears, maybe "There is no function ... anymore"?